src/TimeNode/index.js

```diff
@@ -44,6 +45,23 @@ const timenode = async (options, program) => {
     }
   });
 
+  const statsDb = new loki('stats.json', {
+    adapter: new lfsa(),
+    autoload: true,
+    autoloadCallback: () => {
+      // LokiJS stores BN objects as a string.
+      // This causes problems when loading from persistent storage.
+      // Convert any stored non-BN into BN.
+      const stats = statsDb.getCollection('timenode-stats').data;
```

Comment:
Where exactly the timenode-stats files is stored? Maybe it should be in ~ directory next to othet .files

Reply:
That's a good question, especially when you consider linux based systems and windows. I have not good answer now :|

Reply:
@kosecki123 as @lsaether said, the file is stored in whichever directory the TimeNode process is started from. I agree we should change it to an absolute path.